shared/components/UserDataForm/UserDataForm.helpers.js

```diff
@@ -2,6 +2,6 @@ import * as Crypto from 'expo-crypto';
 
 export const getHashedPassword = async (password) =>
```

Comment:
comme dit dans la PR sign-in, pourquoi pas déplacer cette fonction dans qque chose de plus global car elle est souvent utilisée

Reply:
Ce sera utilisé une seule fois dans le projet et techniquement j'utilise juste une fonction donc c'est généralisé.
Je l'ai mis dans un helper car je ne mets jamais de fonctionnel pur dans mes composants.

Reply:
Fix dans sign in